multus_sriov_cni.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash -x
+
+export RECLONE=${RECLONE:-true}
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+export PATH=/usr/local/go/bin/:/opt/cni/bin/:/home_stack/cni/cni/cnitool/$PATH
+export CNI_PATH=/opt/cni/bin/
+
+MULTUS_CNI_BRANCH=${MULTUS_CNI_BRANCH:-master}
+MULTUS_CNI_REPO=${MULTUS_CNI_REPO:-https://github.com/intel/multus-cni}
+
+SRIOV_CNI_BRANCH=${SRIOV_CNI_BRANCH:-master}
+SRIOV_CNI_REPO=${SRIOV_CNI_REPO:-https://github.com/intel/sriov-cni}
+
+PLUGINS_BRANCH=${PLUGINS_BRANCH:-master}
+PLUGINS_REPO=${PLUGINS_REPO:-https://github.com/containernetworking/plugins.git}
+
+SRIOV_NETWORK_DEVICE_PLUGIN_REPO=${SRIOV_NETWORK_DEVICE_PLUGIN_REPO:-https://github.com/intel/sriov-network-device-plugin}
+SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH=${SRIOV_NETWORK_DEVICE_PLUGIN_BRANCH:-remotes/origin/feature/device_selectors}
+
+export GOROOT=${GOROOT:-/usr/local/go}
+export GOPATH=${WORKSPACE}
+export CNI_BIN_DIR=${CNI_BIN_DIR:-/opt/cni/bin/}
+export CNI_CONF_DIR=${CNI_CONF_DIR:-/etc/cni/net.d/}
+export ALLOW_PRIVILEGED=${ALLOW_PRIVILEGED:-true}
+export NET_PLUGIN=${NET_PLUGIN:-cni}
+
+export KUBE_ENABLE_CLUSTER_DNS=${KUBE_ENABLE_CLUSTER_DNS:-false}
+export API_HOST=$(hostname).$(hostname -y)
+export API_HOST_IP=$(hostname -I | awk '{print $1}')
+export KUBECONFIG=${KUBECONFIG:-/var/run/kubernetes/admin.kubeconfig}
+
+#TODO add autodiscovering
+export MACVLAN_INTERFACE=${MACVLAN_INTERFACE:-eno1}
+export SRIOV_INTERFACE=${SRIOV_INTERFACE:-enp3s0f0}
+
+echo "Working under $WORKSPACE folder"
+mkdir -p $WORKSPACE
+mkdir -p $LOGDIR
+mkdir -p $ARTIFACTS
+mkdir -p $CNI_CONF_DIR
+mkdir -p $CNI_BIN_DIR
+
+pushd $WORKSPACE
+
+
+function configure_multus {
+    echo "Configure Multus"
+    interface=$1
+
+    kubectl delete -f $WORKSPACE/multus-cni/images/multus-daemonset.yml
+    kubectl create -f $WORKSPACE/multus-cni/images/multus-daemonset.yml
+    sleep 60
```

Comment:
` kubectl create -f $WORKSPACE/multus-cni/images/multus-daemonset.yml`
Should be after deploying k8s not before, we are not sure if there is a previous k8s runnig